skrf/network.py

```diff
@@ -539,6 +539,39 @@ def __pow__(self, other: 'Network') -> 'Network':
         else:
             return cascade(self, other)
 
+    def __rshift__(self, other: 'Network') -> 'Network':
+        """
+        Cascade two 4-port networks with "1=>2/3=>4" port numbering.
+
```

Comment:
I suggest adding a connection diagram in the docstring, like :
```suggestion
    Note
    ----
    connection diagram::

              A               B
           +---------+   +---------+
          -|0        1|---|0        1|-
          -|2        3|---|1        3|-
          ...       ... ...       ...
          -|2N-4  2N-3|---|2N-4  2N-3|-
          -|2N-2  2N-1|---|2N-2  2N-1|-
           +---------+   +---------+

```